docs/source/graph_composition.rst

```diff
@@ -21,82 +21,84 @@ Simple composition of operations
 
 The simplest use case for ``compose`` is assembling a collection of individual operations into a runnable computation graph.  The example script from :ref:`quick-start` illustrates this well::
 
-   from operator import mul, sub
-   from graphkit import compose, operation
+   >>> from operator import mul, sub
+   >>> from graphkit import compose, operation
 
-   # Computes |a|^p.
-   def abspow(a, p):
-      c = abs(a) ** p
-      return c
+   >>> # Computes |a|^p.
+   >>> def abspow(a, p):
+   ...    c = abs(a) ** p
+   ...    return c
 
-   # Compose the mul, sub, and abspow operations into a computation graph.
-   graph = compose(name="graph")(
-      operation(name="mul1", needs=["a", "b"], provides=["ab"])(mul),
-      operation(name="sub1", needs=["a", "ab"], provides=["a_minus_ab"])(sub),
-      operation(name="abspow1", needs=["a_minus_ab"], provides=["abs_a_minus_ab_cubed"], params={"p": 3})(abspow)
-   )
+   >>> # Compose the mul, sub, and abspow operations into a computation graph.
+   >>> graphop = compose(name="graphop")(
```

Comment:
That was tough!

Diagrams now include the name of the netop in a label, and `pyplot` has a bug (pydot/pydot#111) when that is a *DOT-keyword* (`graph`, `node`, `edge`), and fails without escaping it.  Unfortunately, the subprocess code does not explain what is happening. 
It would be hard for a user to discover the root of the failure alone, so i choose to avoid that name altogether.

Any fix in the future when its 111 is fixed should be a trivial search'n-replace, since this term is not used anywhere else.
Till then, do you prefer another name (e.g `netop`)?

Reply:
Do you mean if you give the graph the name "graph" then dot throws an exception?  If that is the case I'm fine with choosing whatever name is appropriate for the graph (i.e. "sample_graph").

But as far as the *variable* `graph` is concerned, I would prefer  that the documentation uses "graph" for the variable name.

Reply:
`pipeline`?